node/plan.sh

```diff
@@ -28,7 +28,7 @@ do_prepare() {
 
 do_build() {
   # use --without-snapshot, because https://github.com/nodejs/node/issues/4212
```

Comment:
I know this is nitpicking but we should remove this line if we are no longer using --without-snapshot